--- linux-6.1.24_orig/drivers/spi/spi-sun6i.c	2023-04-13 09:55:40.000000000 -0500
+++ linux-6.1.24/drivers/spi/spi-sun6i.c	2025-04-13 14:55:31.076198383 -0500
@@ -699,6 +699,8 @@
 static const struct of_device_id sun6i_spi_match[] = {
 	{ .compatible = "allwinner,sun6i-a31-spi", .data = (void *)SUN6I_FIFO_DEPTH },
 	{ .compatible = "allwinner,sun8i-h3-spi",  .data = (void *)SUN8I_FIFO_DEPTH },
+	{ .compatible = "allwinner,sun50i-r329-spi", .data = (void *)SUN8I_FIFO_DEPTH },
+	{ .compatible = "allwinner,sun20i-d1-spi",   .data = (void *)SUN8I_FIFO_DEPTH },
 	{}
 };
 MODULE_DEVICE_TABLE(of, sun6i_spi_match);
